### PR TITLE
feat: update, colorize and polish sandbox

### PR DESCRIPTION
**Pull request by santdeleon**
Inspired by @bfriel , I decided to buff up the sandbox code. Thought it'd be cool to have a really solid sandbox with professional quality code so 🤷‍♀️ . Here's a list of some of the major changes. 

**Please verify that everything in the sandbox works as expected. The changes are a nice improvement but not necessary so yeah**

1. Break components out into separate files so that developers can focus on important logic in a digest-able fashion
2. Colorize the transaction console
3. Update the grid to use flex-box
4. Make the transaction console overflow on the y axis and break words
5. Make console information more useful and easy to parse
6. Add welcome messages when the console is empty
7. Add a tag at the bottom of the page "Made with <3 by the Phantom team"
8. Import the Phantom logo and make it a link
9. Add the ability to clear logs
10. Update metadata
11. Use `styled-components`
12. Make dedicated `utils` for some of the core logic
13. Accessibility++. Now you can tab through and interact with the sandbox (with visual queues)
14. Fix the polling for `signAndSendTransaction` and allow 30 seconds for `signAndSend`

https://user-images.githubusercontent.com/22578527/177019149-f7e6dbff-d985-4042-af62-688c03522500.mov

https://user-images.githubusercontent.com/22578527/177019173-70b16db5-206d-49ae-86e1-042d515bede5.mov

Screen Shot 2022-07-02 at 7 52 45 PM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,20 @@
-node_modules
\ No newline at end of file
+# Debugging
```
Comment (santdeleon) [inline .gitignore:1]: maybe overkill for all these?

File: package.json [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
-    "starter"
+    "phantom",
+    "phantom wallet",
+    "phantom-wallet",
+    "codesandbox",
+    "solana",
+    "ethereum",
+    "crypto",
+    "blockchain",
+    "love"
```
Comment (santdeleon) [inline package.json:15]: 👀

File: public/index.html [hunk 3]
```diff
@@ -1,43 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
 	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<meta name="description" content="Phantom Wallet CodeSandbox">
+	<meta name="keywords" content="Phantom, Phantom Wallet, Phantom-Wallet, CodeSandbox, Solana, Ethereum, Crypto, Blockchain, Love">
```
Comment (santdeleon) [inline public/index.html:6]: 👀 ❤️

**Comment by santdeleon** [comment]
Honestly, it would be dope to add animation and pretty UI. We should allot some time to making this sandbox look `fireeeee`

**Comment by santdeleon** [comment]
Okeey doke. Ending it here. I think I've been OCD enough.

**Review by bfriel — APPROVED** [review]
🐐

**Review by jozanza — APPROVED** [review]

**Pull request merged**